src/hotspot/share/logging/logFileStreamOutput.cpp

```diff
@@ -86,19 +87,49 @@ class FileLocker : public StackObj {
   }
 };
 
+static bool write_error_is_shown = false;
```

Comment:
Can you make this a static member of LogFileStreamOutput please?

Reply:
I also wonder whether this should not be a non-static member. Are there many streams? Isn't there one per output destination? If we stream to multiple files, we may want to see the error in each of the files.

Reply:
Good catch! It should be a member of `LogFileStreamOutput`. I fixed it.